src/logic/policies/data.ts

```diff
@@ -117,5 +117,9 @@ async function loadPolicies(
 }
 
 export function isActive(policy: Policy) {
+  return isBefore(new Date(), policy.endTime);
```

Comment:
Mhh, this is a change in logic. Is it correct?

Reply:
Hmm this is new logic for policies. Are you maybe thinking of claims?

Reply:
Mhh, from the diff I see that previously the `isActive` function was implemented as `isBefore(new Date(), addDays(policy.endTime, 3));` but now it's `isBefore(new Date(), policy.endTime)`

Reply:
Ah right (I see you were looking at a commit vs the end result). So a policy is considered inactive once it is past its end time, but you can still create a claim for another 72 hours after that (this logic moved to "canCreateClaim()").